conserver: add advanced serial console package

bmork opened the pull request:
Conserver is an application that allows multiple users to use
serial consoles at the same time, with logging.  It has a client-
server design which makes it easy to manage a distributed set
of serial consoles.  Logging makes post-crash analyses easier.

Signed-off-by: Bjørn Mork <bjorn@mork.no>


Maintainer: @bmork
Compile tested: mvebu, WRT1900ACv1, LEDE master + 17.01
Compile tested: ar71xx, UniFi-AC-PRO, LEDE 17.01.1
Run tested:  mvebu, WRT1900ACv1, LEDE-17.01.1
Run tested: ar71xx, UniFi-AC-PRO, LEDE 17.01.1

hauke commented on net/conserver/files/conserver.init:
```diff
@@ -0,0 +1,17 @@
+
+reload() {
+	service_reload /usr/sbin/conserver
+}
```
Why not use a simple procd init script?

hauke commented on net/conserver/Makefile:
```diff
@@ -0,0 +1,75 @@
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/console $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/etc/$(PKG_NAME)
+	$(INSTALL_CONF) ./files/$(PKG_NAME).cf $(1)/etc/$(PKG_NAME)/
+	$(INSTALL_CONF) ./files/$(PKG_NAME).passwd $(1)/etc/$(PKG_NAME)/
```
I would prefer to touch create the file here instead of shipping an empty file.

bmork commented:
Hauke Mehrtens <notifications@github.com> writes:

> Why not use a simple procd init script?

Good question. I don't know what I'm doing, so I just copied some script
I found :-)

Fixed now


> I would prefer to touch create the file here instead of shipping an empty file. 

Tried to fix that as well.  If I understood you correctly

Replaced all the unreadable $PKG_NAME references in the Makefile at the
same time.
